lib/tools/adb-emu-commands.js

```diff
@@ -5,42 +5,46 @@ import path from 'path';
 
 let emuMethods = {};
 
-emuMethods.isEmulatorConnected = async function (udid) {
-  let emulators = await this.getConnectedEmulators();
-  if (!emulators.length) {
-    return false;
+emuMethods.getEmulators = async function() {
```

Comment:
hmm hmm. we can start with that and see. i would have said lower? what's the cost of running the call?